FeedMe/FeedMe/DayMenuViewController.swift

```diff
@@ -56,4 +57,16 @@ class DayMenuViewController: UIViewController,  UITableViewDelegate, UITableView
         
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        
+        let storyboardName  = Bundle.main.object(forInfoDictionaryKey: "UIMainStoryboardFile")
+        let storyBoard = UIStoryboard(name: storyboardName as! String, bundle: Bundle.main)
+        let detailVc : MenuDetailViewController = storyBoard.instantiateViewController(withIdentifier: "menuDetailVC") as! MenuDetailViewController
+        let currentProduct = self.dayMenuItems[indexPath.row]
```

Comment:
You could add a guard to ensure that you won't get a Index out of bound exception.